src/SelectQuery.php

```diff
@@ -256,10 +256,15 @@ protected function generateJoins(Identifier $identifier): Iterator
     protected function generateOrderBy(Identifier $identifier): Iterator
     {
         foreach ($this->orderBy as $sort) {
+
+			$escapedIdentifier = $sort[0] instanceof Expression
+				? $sort[0]->sql()
+				: $identifier->escapeQualified($sort[0]);			
```

Comment:
Instead of doing this, I would recommend modifying `Identifier` similar to https://github.com/shadowhand/latitude/blob/4505c43f4869d20fd18728c689c9e86153ebcc4b/src/Identifier.php#L71-L72